src/app/components/CustomBadge/CustomBadge.js

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from 'react'
+import { Badge } from 'antd'
+import { CheckOutlined } from '@ant-design/icons'
+// import PropTypes from 'prop-types'
+// import { useTranslation } from 'react-i18next'
+
+function CustomBadge(props) {
+  // const { WRITE_PROPS_HERE } = props
+  // const { ADDITIONAL_DESTRUCTURING_HERE } = user
+
+  // [ADDITIONAL HOOKS]
+  // const { t } = useTranslation('translation')
+  // const { currentLanguage } = t
+
+  // [COMPONENT STATE HOOKS]
+  // const [state, setState] = useState({})
+
+  // [COMPUTED PROPERTIES]
+
+  // [CLEAN FUNCTIONS]
+
+  // [USE_EFFECTS]
+  useEffect(() => {
+    let isComponentMounted = true
+
+    // [EFFECT LOGIC]
+    // write code here...
+    // code sample: isComponentMounted && setState(<your data for state updation>)
+
+    // [CLEAN UP FUNCTION]
+    return () => {
+      // [OTHER CLEAN UP-S (UNSUBSCRIPTIONS)]
+      // write code here...
+
+      // [FINAL CLEAN UP]
+      isComponentMounted = false
+    }
+  }, [])
+
+  return (
+    <Badge
+      count={
+        <CheckOutlined
+          style={{
```

Comment:
Is it possible to configure icon styles with classes? If it's not, please use constant styles